cx_Freeze/finder.py

```diff
@@ -605,8 +607,7 @@ def IncludeFile(self, path, moduleName = None):
             moduleName = name
         info = (ext, "r", imp.PY_SOURCE)
         deferredImports = []
-        module = self._LoadModule(moduleName, open(path, "U"), path, info,
-                deferredImports)
+        module = self._LoadModule(moduleName, 0, path, info, deferredImports)
```

Comment:
What's the reason for passing 0 instead of open(path, "U") for the file handle? Is this intended?

Reply:
In _LoadModule, when the type is imp.PY_SOURCE like this case, the source in the path is always open, then this is leaked. While debugging the optimize, I see this. I'm looking ahead, in the py39 open with "U" is not permitted, the I removed this unnecessary code at this opportunity.

Reply:
So you're saying that argument is unused? If so, perhaps it could be changed to `None`?

Reply:
yes, in _LoadModule, if the type is PY_SOURCE, it always open the source.
Changed to none.